### ISSUE TITLE
section/session feedback form

### ISSUE THREAD

**Issue by ctb** [ngs-docs/angus#193]
do we want to link in session feedback forms? if so how do we avoid unconscious bias issues?

@lexnederbragt has a nice formula "a wish and a star" - we could ask attendees who are giving feedback on a session to say at least one nice thing (a star) as well as one "wish" for change.

I would also say that only a few of us could/should read the feedback and then summarize to the instructors. this can be a lot of work, though, and in the past I've never gotten to actually doing it.

@karenword ?

**Comment by ctb** [comment]
I was thinking a google doc

**Comment by karenword** [comment]
I've created a shared Google Drive folder for sharing assessment data. That is located here:

https://drive.google.com/open?id=0B9AnijVJ1Siba3VITENfczVEVVE

I set up a quick feedback form within that folder for a star and a wish. Let me know if you'd like to see changes! Or make a duplicate and change it yourself if that's easier : ).  Link to the user-facing page is here:
https://docs.google.com/a/ucdavis.edu/forms/d/e/1FAIpQLSc8wlIVtbh7XQdLAZSrfyy9ExkC6zMFv18gx8PfjSfzIo6aOg/viewform?usp=sf_link
